observability/tracing.go

```diff
@@ -171,29 +173,74 @@ func getExporter(collectorURL string) (sdktrace.SpanExporter, error) {
 					)
 				} else {
 					log.Debugf("Creating OpenTelemetry collector with URL https://%s", collectorURL)
-
-					client = otlptracehttp.NewClient(otlptracehttp.WithEndpoint(collectorURL),
-						tracingOptions,
-					)
+					// That's mainly for testing
+					if tracingOpt.Otel.SkipVerify {
+						client = otlptracehttp.NewClient(otlptracehttp.WithEndpoint(collectorURL),
```

Comment:
Since the comment says this is really mainly for testing, do we want to log a message at TRACE level here to let the user know if they configured it in an insecure way? Something like:
```go
log.Trace("OpenTelemetry collector will not verify the remote certificate")
```